dev/com.ibm.ws.jsp.2.3_fat/fat/src/com/ibm/ws/jsp23/fat/tests/JSPChannelTest.java

```diff
@@ -192,8 +193,9 @@ public void testIgnoreWriteAfterCommitFalse() throws Exception {
             Assert.assertFalse("The redirect failed!", containsMessage);
```

Comment:
Shouldn't the message be here "The redirect was successful"? We're checking that the redirect message was not contained in the response, if it is found then the redirect was successful when it should not have been.